player/lua/stats.lua

```diff
@@ -116,6 +116,17 @@ local function compat(p)
 end
 
 
+local function bytestostring(b)
```

Comment:
Maybe such formatting functions shouldn’t be duplicated everywhere? There’s at least one instance of humanized size formatting in osc.lua, too.

Reply:
Really? I only saw this "on the fly formatting" here: https://github.com/wm4/mpv/blob/master/player/lua/osc.lua#L1844 and didn't want to do it like that, hence this function.
There's one more problem with all of this: the OSC uses KiB/MiB units, while get_*_osd property getters return formatted numbers with Mb/Kb units. Internally, mpv also has no general formatting for this: https://github.com/mpv-player/mpv/blob/master/player/command.c#L323 https://github.com/mpv-player/mpv/blob/master/player/osd.c#L267

Reply:
At least osc.lua and stats.lua could share that function though.

Reply:
Like that?